src/coreclr/jit/emitarm64.cpp

```diff
@@ -8761,6 +8827,84 @@ void emitter::emitIns_R_R(instruction     ins,
             fmt = IF_SVE_GK_2A;
             break;
 
+        case INS_sve_frecpe:
+        case INS_sve_frsqrte:
+            assert(insScalableOptsNone(sopt));
+            assert(insOptsScalableStandard(opt));
+            assert(opt != INS_OPTS_SCALABLE_B); // reserved
```

Comment:
Nit: We can check that `opt` is one of the standard scalable types, but not `B`, with `insOptsScalableAtLeastHalf`.

Reply:
Yea that is the proper way to do it. Sometimes I forget since I don't know all the rules for the instruction immediately :)